alarm/utils.py

```diff
@@ -33,3 +33,55 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
             return True
 
     return False
+
+
+def create_thresholds_brakes_from_recent_candles_update(trade_pair):
+    thresholds = Threshold.objects.filter(trade_pair=trade_pair)
+    for threshold in thresholds:
+        ThresholdBrake.objects.create(threshold=threshold)
+
+
+def create_message_for_threshold_break(trade_pair):
+    three_second_ago = timezone.now() - timedelta(seconds=3)
+
+    messages = {}
+
+    trade_pair_str = format_trade_pair_for_message(trade_pair)
+
+    threshold_brake_prices = list(
+        ThresholdBrake.objects.filter(threshold__trade_pair=trade_pair, happened_at__gte=three_second_ago).values_list(
+            'threshold__price', flat=True))
+
+    threshold_brake_prices_str = ', '.join([f'{price}$' for price in threshold_brake_prices])
+
+    last_candle = Candle.objects.filter(trade_pair=trade_pair).order_by('-modified').last()
+    last_candle_high_price = last_candle.high_price if last_candle else None
+
+    message = messages.setdefault(trade_pair,
+                                  f" Attention! Trade pair {trade_pair_str} has broken thresholds ")
+    message += f'{threshold_brake_prices_str} and current price is {last_candle_high_price}$ '
+    messages[trade_pair] = message
+    return ''.join(messages.values())
+
+
+def combine_threshold_break_messages(trade_pair):
```

Comment:
Naming is invalid. Why?

Reply:
The function name does not correctly reflect the purpose or functionality of the function. If answering the question "What do we need this function for", then the more correct name is "create_spoken_alert_for_threshold_break".